### ISSUE TITLE
Check for accessability issues

### ISSUE THREAD

**Issue by damianstasik** [opentofu/opentofu.org#59]
The website and docs should be as accessible as possible, we have a few things already done to improve the support in screen readers, but we should run every page through an accessibility checker and work on whatever will be reported.

**Comment by janosdebugs** [comment]
I've done this and opened separate issues, see #259 and 260
